packages/cli/src/cmds/cmd_build/generate.rs

```diff
@@ -1,8 +1,11 @@
-use std::path::Path;
-
-use cu::Result;
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2025 Megaton contributors
 
+use std::path::Path;
+use cu::{Result, Context, Command, Spawn, lv, Child};
```

Comment:
The recommended style from cu (i.e. from me) is to import prelude only , see https://cu.pistonite.dev/pistonite_cu/#general-principal

Also try to keep the imports organized in groups (std -> libs -> super/crate)
IIRC cargo fmt also requires the imports sorted in each group (groups are separated by empty line)